src/util/temp-dir.js

```diff
@@ -127,7 +127,7 @@ export class TempDir {
       return;
     }
     log.debug(`Removing temporary directory: ${this.path()}`);
-    this._removeTempDir && this._removeTempDir();
+    return this._removeTempDir && this._removeTempDir();
```

Comment:
Use `&& promisify(this._removeTempDir())` here, for the reason explained above.

Reply:
I forgot: Add `()` after `promisify(...)` to actually call the function returned by `_removeTempDir()`.